### DISCUSSION TITLE
forgot syncthing password

### DISCUSSION THREAD

**Discussion by serj-maks** [Martchus/syncthingtray#215] [Q&A] [answered]
how i can reset my password ?

**Comment by Martchus** [comment]
The password is set via Syncthing itself (and not Syncthing Tray). So I suppose you just stop Syncthing itself, edit its config to remove the password and start it again.

**Reply by Martchus** [reply, marked as answer]
Note that Syncthing Tray doesn't even use the password; it uses the API key. If you're opening Syncthing itself via Syncthing Tray's built-in web view you also might not need the password (but I'm not sure about this after recent changes in Syncthing itself). So if that's still working you should also be able to set the password this way.

**Reply by serj-maks** [reply]
thank you ! 1. stop sync-tray 2. go to syncthing localhost. (now I just log in - without password) 3. copy API key 4. paste it to sync-tray. For me it works
